backend/config/settings/production.py

```diff
@@ -71,8 +71,8 @@
 GRAPHENE["MIDDLEWARE"] += ["config.sentry.middleware.SentryMiddleware"]  # noqa
 
 # Sentry
-SENTRY_DSN = cast(str, env("SENTRY_DSN"))
-SENTRY_LOG_LEVEL = cast(int, env.int("DJANGO_SENTRY_LOG_LEVEL", logging.INFO))
+SENTRY_DSN: str = env("SENTRY_DSN")
+SENTRY_LOG_LEVEL: int = cast(int, env.int("DJANGO_SENTRY_LOG_LEVEL", logging.INFO))
```

Comment:
```suggestion
SENTRY_LOG_LEVEL: int = env.int("DJANGO_SENTRY_LOG_LEVEL", logging.INFO)
```
Unnecessary cast?

Reply:
No, the Django Environ package is poorly type annotated, and does not return the proper type. Without casting it, the IDE will yell at us for claiming that it's an `int`.